test/shared/src/main/scala/zio/test/Assertion.scala

```diff
@@ -59,6 +59,12 @@ class Assertion[-A] private (val render: Render, val run: (=> A) => AssertResult
   override final def hashCode: Int =
     toString.hashCode
 
+  /**
+   * Labels this assertion with the specified string.
+   */
+  final def label(string: String): Assertion[A] =
+    new Assertion(infix(param(self), "@@", param(quoted(string))), run)
```

Comment:
Might suggest `??` instead of `@@` to match ZIO Config; and also have `??` operator alias.

Reply:
Great! Will change now.